pkg/await/retry.go

```diff
@@ -41,7 +56,7 @@ func (r *retrier) Do() error {
 	for r.tries <= r.maxRetries {
 		err = r.try(r.tries)
 		r.tries++
-		if errors.IsNotFound(err) {
+		if errors.IsNotFound(err) || meta.IsNoMatchError(err) {
```

Comment:
I assume this is for the CRDs. This kind of scares me. Is client-go changing the error API? And if so, how do we know what else to check?

Reply:
Yes, this is required for the RESTMapper, which will return an unknown Kind error if the relevant CRD hasn't been registered in the cache yet.